Create array_search() doc #3047

reganmarie opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

- Closes issue #3047 
- Created array_search.md on file path: `docs/content/php/concepts/arrays/terms/array-search/array-search.md`
- Added Title, Description, Subjects, Tags, and Catalog Content
- Added Description, Syntax, Example, and Codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=3161) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=3161) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=3161) it.</sub>

caupolicandiaz commented on content/php/concepts/arrays/terms/array-search/array-search.md:
````diff
@@ -0,0 +1,54 @@
+```
+
+- `value`: The value to search for.
+> **Note:** String values are case-sensitive for this function.
````
I would suggest moving the note after all the parameters

reganmarie replied:
Fixed!

caupolicandiaz commented on content/php/concepts/arrays/terms/array-search/array-search.md:
```diff
@@ -0,0 +1,54 @@
+  - 'paths/computer-science'
+---
+
+The **`array_seach()`** function searches an array for a given value and returns the first matching key for that value.
```
```suggestion
The **`array_search()`** function searches an array for a given value and returns the first matching key for that value.
```

caupolicandiaz reviewed (changes requested):
@reganmarie Just added a couple of comments. Please update, and tag me when you've finished making the changes. Thanks

reganmarie commented:
Fixed the spelling of search in line 16 and moved the note under all of the parameters! @caupolicandiaz - I realized that you may want the link directly for the newest commit :) https://github.com/Codecademy/docs/pull/3161/commits/98058b39ffd6cc52da523fd0505c43d69c35bc7d

caupolicandiaz reviewed (approved):
@reganmarie Looks good, I just made a couple of minor formatting edits directly. This is ready for a second review.

reganmarie commented:
Wonderful, thank you @caupolicandiaz ! Please let me know if there's anything else I can do in the meantime :)

SSwiniarski reviewed (approved):
@reganmarie Looks good.

reganmarie commented:
Thank you @SSwiniarski ! :)
